Sources/App/Controllers/PostController.swift

```diff
@@ -12,8 +12,8 @@ final class PostController: ResourceRepresentable {
 
     /// When consumers call 'POST' on '/posts' with valid JSON
     /// create and save the post
-    func create(request: Request) throws -> ResponseRepresentable {
-        let post = try request.post()
+    func create(req: Request) throws -> ResponseRepresentable {
```

Comment:
Using `_ req: Request` here would be preferred. `Request` is strongly typed so we don't need to be redundant with the parameter label.